src/modules/ekf2/EKF/velocity_fusion.cpp

```diff
@@ -79,12 +82,61 @@ void Ekf::updateVelocityAidSrcStatus(const uint64_t &time_us, const Vector3f &ob
 	aid_src.timestamp_sample = time_us;
 }
 
+void Ekf::fuseBodyVelocity(estimator_aid_source3d_s &aid_src)
+{
+	Vector3f innov;
+	Vector3f innov_var;
+	matrix::Matrix<float, 1, 23UL> H;
+	bool fused[3] = {false, false, false};
+	const auto state_vector = _state.vector();
+	Vector3f R(aid_src.observation_variance);
+
+	sym::ComputeEvBodyVelVarAndHx(state_vector, P, &innov_var, &H);
+	innov = _R_to_earth.transpose() * _state.vel - Vector3f(aid_src.observation);
+	innov.copyTo(aid_src.innovation);
+	(innov_var + R).copyTo(aid_src.innovation_variance);
+	const float innov_gate = math::max(_params.ev_vel_innov_gate, 1.f);
+	setEstimatorAidStatusTestRatio(aid_src, innov_gate);
+
+	if (aid_src.innovation_rejected) {
+		return;
+	}
+
+	for (uint8_t index = 0; index <= 2; index++) {
+		if (index == 1) {
+			sym::ComputeEvBodyVelVarAndHy(state_vector, P, &(innov_var(index)), &H);
+
+		} else if (index == 2) {
+			sym::ComputeEvBodyVelVarAndHz(state_vector, P, &(innov_var(index)), &H);
+		}
+
+		Vector3f meas_pred = _R_to_earth.transpose() * _state.vel;
+		innov_var(index) += aid_src.observation_variance[index];
```

Comment:
Just make another temporary variable to be more explicit with the names, it doesn't cost anything.